frontend/packages/ceph-storage-plugin/src/features.ts

```diff
@@ -139,6 +142,18 @@ export const detectOCS: FeatureDetector = async (dispatch) => {
   }
 };
 
+export const detectManagedODF: FeatureDetector = async (dispatch) => {
+  try {
+    const ns = await k8sGet(NamespaceModel, CEPH_STORAGE_NAMESPACE);
+    if (ns) {
+      const isManagedCluster = ns?.metadata?.labels?.[ODF_MANAGED_LABEL];
```

Comment:
Label added to the `openshift-storage` ns for ODF managed services: https://gitlab.cee.redhat.com/service/managed-tenants/-/merge_requests/1100